content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -428,8 +422,14 @@ that have a different value and are owned by another manager will result in a
 collaborator's changes. Conflicts can be forced, in which case the value will be
 overridden, and the ownership will be transferred.
 
-It is meant both as a replacement for the original `kubectl apply` and as a
-simpler mechanism to write controllers.
+If you remove a field from a configuration an apply it, server side apply checks
+if there are any other field managers that also own the field.  If the field is
+not found to be owned by any other field managers, it deleted from the live
+object or reset to a default value. The same rule applies to associative list or
+map items.
+
+Server side apply is meant both as a replacement for the original `kubectl
+apply` and as a simpler mechanism to write controllers.
```

Comment:
```suggestion
apply` and as a simpler mechanism for controllers to enact their changes.
```

Although I'm not sure if we should say this yet, as we don't have good library support for it yet...